### PR TITLE
"When nobody wants to remove the max data size limit but only increase it"

### PR DESCRIPTION
**Pull request by Berthalamew**
Fine I'll do it myself
(put "remove_data_size_limit = true" inside the H2Codez.conf to remove the max data size limit in its entirety)

### CODE REVIEW

File: H2Codez/Common/H2EKCommon.cpp [hunk 1]
```diff
@@ -408,6 +408,12 @@ void H2CommonPatches::Init()
 
 	set_tag_data_max_size(0x3000000);
 
+	if (conf.getBoolean("remove_data_size_limit", true))
+	{
+		set_tag_data_max_size(INT_MAX);
```
Comment (num0005) [inline H2Codez/Common/H2EKCommon.cpp:413]: idk dude INT_MAX might be too large, maybe INT_MAX - 5

File: H2Codez/Common/H2EKCommon.cpp [hunk 2]
```diff
@@ -363,7 +363,7 @@ char __cdecl tags__fix_corrupt_fields___hook(tag_block_defintions *def, tag_bloc
 	return tags__fix_corrupt_fields_org(def, data, 1);
 }
 
-static void set_tag_data_max_size(size_t limit)
```
Comment (num0005) [inline H2Codez/Common/H2EKCommon.cpp:366]: but like seriously keep the static

Comment (Berthalamew) [reply H2Codez/Common/H2EKCommon.cpp:366]: The variable won't change after putting the option if it's static?

File: H2Codez/Common/H2EKCommon.cpp (hunk 1 above)
Comment (num0005) [reply H2Codez/Common/H2EKCommon.cpp:413]: default to false

Comment (num0005) [reply H2Codez/Common/H2EKCommon.cpp:413]: and a setting to set make size would be best, dunno if we have a range limited setting option

Comment (Berthalamew) [reply H2Codez/Common/H2EKCommon.cpp:413]: Idk if there are no issues with putting the size limit to its max there isn't really a point of putting a setting for how big you want it in (so far I haven't seen any issues when putting it to MAX)

**Comment by num0005** [comment]
#77 needs to be addressed before this can be merged.
@Berthalamew what's the status on that?

**Review by Kelzorz — COMMENTED** [review]
PR is inside joke.

**Review by num0005 — CHANGES_REQUESTED** [review]
Yep still an inside joke. @Berthalamew needs to stop putting it on his CV.

**Comment by num0005** [comment]
But I could just merged it 🤔

**Pull request closed without merging**